### PR TITLE
test/e2e/framework:move OpenWebSocketForURL to subpackage(new) websocket

### PR DESCRIPTION
**Pull request by tanjunchen**
**What type of PR is this?**
 
 /kind cleanup
 /priority backlog 

**What this PR does / why we need it**:

Here are some reasons to support this PR.

1. It is better to represent its intention for OpenWebSocketForURL with new subpackage websocket. subpackage websocket as its name implies.
2. It is easy to add and maintain code related to websocket in e2e.
3. ref https://github.com/kubernetes/kubernetes/issues/77095

**Which issue(s) this PR fixes**:

ref:https://github.com/kubernetes/kubernetes/issues/77095

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
None
```

### CODE REVIEW

File: test/e2e/common/pods.go [hunk 1]
```diff
@@ -44,6 +44,7 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 	e2ekubelet "k8s.io/kubernetes/test/e2e/framework/kubelet"
 	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	websocketutil "k8s.io/kubernetes/test/e2e/framework/websocket"
```
Comment (gavinfish) [inline test/e2e/common/pods.go]: Follow the naming pattern like `e2ewebsocket`

File: test/e2e/framework/websocket/websocket_util.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```
Comment (gavinfish) [inline test/e2e/framework/websocket/websocket_util.go]: Update year to 2020

File: test/e2e/kubectl/portforward.go [hunk 3]
```diff
@@ -39,6 +39,7 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 	e2ekubectl "k8s.io/kubernetes/test/e2e/framework/kubectl"
 	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	websocketutil "k8s.io/kubernetes/test/e2e/framework/websocket"
```
Comment (gavinfish) [inline test/e2e/kubectl/portforward.go]: Same as above

File: test/e2e/common/pods.go (hunk 1 above)
Comment (oomichi) [reply test/e2e/common/pods.go]: +1
It is nice to add it into `./hack/.import-aliases` also.

**Review by oomichi — COMMENTED** [review]
Most part seems good for me.
just one comment.

File: test/e2e/common/pods.go (hunk 1 above)
Comment (tanjunchen) [reply test/e2e/common/pods.go]: sure i will update it

Comment (k-toyoda-pi) [reply test/e2e/common/pods.go]: There are 2 alias name pattern in e2e. `e2efoo` or `e2efooutil`. Which is better?

File: test/e2e/common/pods.go [hunk 4]
```diff
@@ -44,6 +44,7 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 	e2ekubelet "k8s.io/kubernetes/test/e2e/framework/kubelet"
 	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	e2ewebsocketutil "k8s.io/kubernetes/test/e2e/framework/websocket"
```
Comment (oomichi) [inline test/e2e/common/pods.go]: It is better to set the alias name as `e2ewebsocket` for consistent name as the above.

Comment (tanjunchen) [reply test/e2e/common/pods.go]: sure.

Comment (tanjunchen) [reply test/e2e/common/pods.go]: done

**Review by oomichi — COMMENTED** [review]
Thanks for updating.

/retest
/lgtm
/approve

**Pull request merged**